install/openshift/cleanup.sh

```diff
@@ -7,6 +7,7 @@ oc delete ns $NS
 oc delete crd alerts.synopsys.com
 oc delete crd blackducks.synopsys.com
 oc delete crd opssights.synopsys.com
+oc delete crd samples.synopsys.com
```

Comment:
remove the sample crd from cleanup script

Reply:
I removed it